lib/stats/DefaultStatsFactoryPlugin.js

```diff
@@ -314,6 +315,22 @@ const { makePathsRelative, parseResource } = require("../util/identifier");
  * @property {ExtractorsByOption<Dependency, StatsModuleTraceDependency>} moduleTraceDependency
  */
 
+/**
+ * @param {string} readableIdentifier user readable identifier of the module
+ * @param {Module} module base module type
+ * @returns {string} an elided readableIdentifier, when readableIdentifier exceeds a maximum length
+ */
+const truncateLongExternalModuleReadableIdentifier = (
+	readableIdentifier,
+	module
+) => {
+	const maxLength = 80;
+	return module instanceof ExternalModule &&
+		readableIdentifier.length > maxLength
+		? readableIdentifier.substring(0, maxLength) + "..."
```

Comment:
```suggestion
		? readableIdentifier.substring(0, maxLength) + "...(truncated)"
```

cc @alexander-akait I think its more user friendly if its _blatantly obvious_ that truncation is happening. The ellipsis alone is not clear enough.

Reply:
Agree, let's add `(truncated)`

Reply:
@TheLarkInn , @alexander-akait Thanks for your feedback on this pull request.  In the original issue, [16475](https://github.com/webpack/webpack/issues/16475), Zack suggested an alternate way to achieve my goal using delegated modules.  I have used his idea, and it is sufficient for my needs.  As a result, I'm not going to continue with this pull request.  If you think it has value outside of my needs, feel free to make the changes that you suggested. Otherwise, please feel free to close the pull request.

Reply:
Sure. I'll finish it.